src/main/java/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -145,7 +145,7 @@ public void removeSurfaceFluids(Claim exclusionClaim)
 					Block block = lesser.getWorld().getBlockAt(x, y, z);
 					if(exclusionClaim != null && exclusionClaim.contains(block.getLocation(), true, false)) continue;
 					
-					if(block.getType() == Material.STATIONARY_LAVA || block.getType() == Material.WATER || block.getType() == Material.STATIONARY_WATER || block.getType() == Material.LAVA)
+					if(block.getType() == Material.LAVA || block.getType() == Material.WATER)
```

Comment:
So both stationary and flowing variants are now considered one material?

Reply:
yes